### PR TITLE
Homepage and loginsignup added

### PR DESCRIPTION
**Pull request by MonikaJhingan**
Please review urgently.

### CODE REVIEW

**Review by vishalg8454 — APPROVED** [review]
Looks goods to merge.

**Review by srijani07 — APPROVED** [review]
Looks good to merge. 
Suggestion: Add a screenshot of the page in the PR description so that it becomes easier to review.

File: Authentication/Signup/signup.css [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+  .show-password{
+      font-size: 0.5rem;
+  }
+
+  .show-password-checkbox{
+      font-size: 1rem;
+  }
+
+  .hover-underline{
+      margin: 8px;
```
Comment (shaikhhamza11) [inline Authentication/Signup/signup.css]: you can use 0.8rem

Comment (MonikaJhingan) [reply Authentication/Signup/signup.css]: done

**Pull request merged**